Dockerfile

```diff
@@ -12,43 +12,34 @@
 # 
 # docker build  --build-arg BUILD_REPO=YourFork/PokemonGo-Bot --build-arg BUILD_BRANCH=6a4580f .
 
-FROM python:2.7.12-alpine
+FROM alpine
```

Comment:
yes but adding python and pip and stuff is the phthon:2.7.12-alpine build I think